pom.xml

```diff
@@ -28,6 +28,11 @@
             <version>${junit-jupiter.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>3.141.59</version>
```

Comment:
just a best practice to extract versions to `<properties/>`

Reply:
Thank you, added!